src/player.js

```diff
@@ -10,6 +10,11 @@ import LoggerFactory from './utils/logger'
 import Html5 from './engines/html5/html5'
 import PluginManager from './plugin/plugin-manager'
 import StateManager from './state/state-manager'
+import TrackTypes from './track/track-types'
```

Comment:
We need to export track types as an enum via the player (like we did with the states and the events).
` get Track(): { [track: string]: string } {
    return TrackTypes;
   }		   }`

When developers would like to get the audio track for example it will look like:
`let audioTracks = player.getTracks(player.Track.AUDIO);`